app/routes/settings/components/LogoutPanel.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { connect } from "react-redux"
+
+import PropTypes from "prop-types"
+import EditItemPanel from "../../shared/components/EditItemPanel"
+import {settingsLogoutUser} from "../../../modules/settings/actions/settings-logout-user"
+
+const LogoutPanel = ({
+  username,
+  logout,
+}) => {
+  return (
+    <EditItemPanel
```

Comment:
Added log out panel which checks electron session. Will be updated in the future so that it is either in the menu or is always shown in the app.

<img width="899" alt="screen shot 2018-07-12 at 4 45 11 pm" src="https://user-images.githubusercontent.com/7866060/42665147-249b52bc-85f3-11e8-950d-2c5d70f33184.png">